crypto/src/mls/client/mod.rs

```diff
@@ -454,14 +515,6 @@ impl Client {
     }
 }
 
-impl PartialEq for Client {
```

Comment:
No reason not to keep implementing `PartialEq` and `Eq` for `Client`:

```rust
let Some(ClientInner{id: sid,..}) = self.state.lock().await.deref() else { return false };
let Some(ClientInner{id: oid,..}) = other.state.lock().await.deref() else { return false };
sid == oid
```

Reply:
I was wrong here: of course we can't `.await` within a non-async function, such as `PartialEq`.